index.html

```diff
@@ -10,74 +10,112 @@
 </head>
 
 <body>
-  <header>
-    <h1>Jelena de la Sauce</h1>
-    <nav class="main-nav">
-      <a href="#contact" class="button">Contact</a>
+  <header class="container-head">
+    <h1 class="child-head-name">Jelena de la Sauce</h1>
+    <nav class="child-head-nav main-nav">
```

Comment:
Hi Jelena, ich hatte den Aufbau des `nav`Tag genauso wie Du - CJ hat mir bei einer Durchsicht meines Codes gesagt, dass es üblich sei, in `nav`eine unordered list `ul`einzufügen und die einzelnen `a`Tags dann in `li`Tags einzupacken

Reply:
Hi Tanja, thank you for checking and also for the advice - didnt know that yet! :-)